linalg/kernels.hpp

```diff
@@ -188,6 +188,35 @@ void Mult(const int height, const int width, TA *data, const TX *x, TY *y)
    }
 }
 
+/** @brief Matrix transpose vector multiplication: y = At x, where the matrix A
+    is of size @a height x @a width with given @a data, while @a x and @a y
+    specify the data of the input and output vectors. */
+template<typename TA, typename TX, typename TY>
+MFEM_HOST_DEVICE inline
+void MultT(const int height, const int width, const TA *data, const TX *x,
+           TY *y)
+{
+   if (width == 0)
+   {
+      for (int row = 0; row < height; row++)
+      {
+         y[row] = 0.0;
+      }
+      return;
+   }
```

Comment:
The matrix is transposed so I think `height` and `width` should be inverted:
```suggestion
   if (height == 0)
   {
      for (int row = 0; row < width; row++)
      {
         y[row] = 0.0;
      }
      return;
   }
```

Reply:
yes!! thank you! i should have been more careful when copy/pasting this part=p